utils/utils_test.go

```diff
@@ -0,0 +1,248 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/spf13/viper"
+
+	"golang.org/x/exp/slices"
+)
+
+func TestGetRunStatuses(t *testing.T) {
+	tests := []struct {
+		includeDeleted bool
+		numStatuses    int
+	}{
+		{includeDeleted: false, numStatuses: 7},
+		{includeDeleted: true, numStatuses: 8},
+	}
+	for _, test := range tests {
+		runStatuses := GetRunStatuses(test.includeDeleted)
+		if len(runStatuses) != test.numStatuses {
+			t.Errorf("Expected %d statuses, got %d", test.numStatuses, len(runStatuses))
+		}
+
+		if test.includeDeleted {
+			if !slices.Contains(runStatuses, "deleted") {
+				t.Errorf("Expected runStatuses to contain deleted")
+			}
+		} else if slices.Contains(runStatuses, "deleted") {
+			t.Errorf("Expected runStatuses not to contain deleted")
+		}
+	}
+}
+
+func TestHasAnyPrefix(t *testing.T) {
```

Comment:
It's great to see many tests being added to the Go client. Can you please also add the code coverage information to in the output? IOW, just edit `Makefile` and use `go test -cover ./...` there, with the option `-cover`.  

(Note also that I think we can remove `-v` in that command, because it will make a nicer formatting with coverage without it, and if some test fails, then the failed test will be printed out in full anyway.)

Reply:
That sounds very good, thanks